packages/apps/job-launcher/server/src/modules/storage/storage.service.ts

```diff
@@ -40,24 +43,20 @@ export class StorageService {
     }
   }
 
-  public async uploadManifest(
-    manifest: FortuneManifestDto | CvatManifestDto | string
+  public async uploadFile(
+    origin: object,
```

Comment:
why do you need both origin and encrypted file? in this method you should not care about if the file is encrypted or not, just upload the file passed